### PR TITLE
Change the field name from defaultBrokerClass to default-broker-class

### PR DESCRIPTION
**Pull request by houshengbo**
Fixes #issue-number or description of the problem the PR solves

## Proposed Changes 

-
-
-

### CODE REVIEW

File: docs/install/operator/configuring-eventing-cr.md [hunk 1]
```diff
@@ -182,5 +180,5 @@ metadata:
   name: knative-eventing
   namespace: knative-eventing
 spec:
-  defaultBrokerClass: MTChannelBasedBroker
+  default-broker-class: MTChannelBasedBroker
```
Comment (matzew) [inline docs/install/operator/configuring-eventing-cr.md:183]: https://github.com/knative/eventing-operator/blob/release-0.14/config/300-eventing-v1alpha1-knativeeventing-crd.yaml#L75

isn't that `defaultBrokerClass` ?

Comment (aliok) [reply docs/install/operator/configuring-eventing-cr.md:183]: Yeah we need to change the field name in the CRD file and in the Go code as well: https://github.com/knative/eventing-operator/blob/release-0.14/pkg/apis/eventing/v1alpha1/knativeeventing_types.go#L75

Comment (aliok) [reply docs/install/operator/configuring-eventing-cr.md:183]: Ok saw https://github.com/knative/eventing-operator/pull/200 now

**Comment by houshengbo** [comment]
/hold
Let's hold it, since we do not change this name for release 0.14.x.

File: docs/install/operator/configuring-eventing-cr.md (hunk 1 above)
Comment (houshengbo) [reply docs/install/operator/configuring-eventing-cr.md:183]: Let's hold it for now, since we do not change it for 14.

**Comment by abrennan89** [comment]
@houshengbo should this still be on hold? Which version will this be active in?

**Comment by evankanderson** [comment]
I think we wanted to go the other way on names.

Going to close this.

/close.

**Pull request closed without merging**